e107_plugins/pm/pm_menu.php

```diff
@@ -102,7 +102,7 @@ function pm_show_popup($pm_inbox, $alertdelay = 0)
 $template = e107::getTemplate('pm', 'pm_menu');
 
 //if(!isset($pm_menu_template))
-if(!isset($PM_MENU_TEMPLATE))
+if(!isset($template))

```

Comment:
pm_menu_template is new, before v2.1.5 theres was no such template, so no chance for legacy themes to use that template...